demos/palm/web/textfx/README.md

```diff
@@ -0,0 +1,110 @@
+# TextFX
+
+By Google Creative Lab
+
+## Contents
+
+- [About](#about)
+- [How it Works](#how-it-works)
+- [Requirements](#requirements)
+- [Developer Setup](#developer-setup)
+- [Contributors](#contributors)
+- [License](#license)
+- [Notes](#notes)
+
+## About
+
+_TextFX_ is an AI experiment built in collaboration with GRAMMY® Award-winning rapper Lupe Fiasco.
+
+It contains 10 AI-powered 10 tools, each designed to explore creative possibilities with text and language.
+
+![TextFX demo gif](https://storage.googleapis.com/textfx-assets/demo.gif)
+
+A live hosted version of _TextFX_ is available at [textfx.withgoogle.com](https://textfx.withgoogle.com/).
+
+This experiment is an example of how you can use the PaLM API to build applications that leverage Google's state of the art large language models (LLMs).
+
+The PaLM API consists of two services, each with a distinct method for generating content:
```

Comment:
> The PaLM API consists of two services

This is false now, and will become increasingly more false as we add more services. No need to list things you don't use - just drop references to Chat if they're irrelevant.

Reply:
Got it! Will update in a separate PR.